data-manipulation/hashing/murmur/hash-data-using-murmur3.yml

```diff
@@ -28,3 +28,21 @@ rule:
         - and:
           - number: 0x87c37b91114253d5 = 64-bit c1 for 128-bit hash
           - number: 0x4cf5ad432745937f = 64-bit c2 for 128-bit hash
+        - and:

```

Comment:
thoughts on adding the XOR part?

Reply:
do you mean adding to the last block `count(mnemonic(xor)): 3`? Should we add this as optional? I think that basic block is already unique enough to conclude this is MurmurHash3. But still a good idea to mark those 🤔

Reply:
I think including it as a (optional) new rule part makes sense, maybe even using the `match` on XOR encode

Reply:
@mr-tz 

> maybe even using the match on XOR encode

Do you mean the `encode data using XOR`? That rule includes `characteristic: tight loop`, which doesn't apply here.

Reply:
Ok, then just XOR (as optional per your discretion)